content/en/real_user_monitoring/guide/monitor-hybrid-react-native-applications.md

```diff
@@ -0,0 +1,323 @@
+
+### Initialize the React Native SDK
+
+In order to initialize the SDK on both the React Native and the native side, you can follow the [React Native Monitoring documentation][1]. 
+
+With this setup, you can call both the native and React Native SDKs for logs, traces and RUM.
+
+We recommend this solution if you haven't used our SDK on the native side before you initialize it on the React Native side.
+
+{{< tabs >}}
+{{% tab "Android" %}}
+On Android, add the Datadog Android SDKs to your dependencies in your `android/app/build.gradle` file:
+
+```java
+// The version is set by @datadog/mobile-react-native
+implementation "com.datadoghq:dd-sdk-android-rum"
+implementation "com.datadoghq:dd-sdk-android-logs"
+implementation "com.datadoghq:dd-sdk-android-trace"
+implementation "com.datadoghq:dd-sdk-android-webview"
+```
+
+{{% /tab %}}
+{{% tab "iOS" %}}
+
+On iOS, add the Datadog iOS SDKs to your dependencies in your ios/Podfile to use in Objective C files:
+
+```ruby
+# Make sure the version matches the one from node_modules/@datadog/mobile-react-native/DatadogSDKReactNative.podspec
+pod 'DatadogSDKObjc', '~> 2.5.0'
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Tracking native RUM views
+
+If you use a navigation library for your React Native app like `react-navigation`, using the `nativeViewTracking` configuration option creates many duplicate views.
+
+If this is the case, track your native RUM Views manually. See documentation for [iOS][2] and for [Android][3].
+
+### Tracking native RUM resources
+
+If you have enabled tracing with your backend, first-party hosts for your native RUM Resources are the same as for your React Native RUM resources.
+
+{{< tabs >}}
+{{% tab "Android" %}}
+
+If you use OkHttp, you can use Datadog's interceptor to [automatically track network requests][1]. Alternatively, you can [manually track resources][2].
+
+[1]: https://docs.datadoghq.com/real_user_monitoring/ios/advanced_configuration/?tab=objectivec#automatically-track-network-requests
+[2]: https://docs.datadoghq.com/real_user_monitoring/android/advanced_configuration/?tab=kotlin#automatically-track-network-requests
+
+{{% /tab %}}
+{{% tab "iOS" %}}
+
+You can track network requests by monitoring your `URLSession`. Learn more about how to [automatically track network requests][3].
+
+[3]: https://docs.datadoghq.com/real_user_monitoring/android/advanced_configuration/?tab=kotlin#custom-resources
+{{% /tab %}}
+{{< /tabs >}}
+
+### Limitations
+
+Make sure your native code is not executed in a moment in the lifecycle where the RN initialization has not been called yet.
+
+When you initialize the SDK on the React Native side (JavaScript / TypeScript) it will also be initialized on the native side. If you write any native code that relies on our SDK, you have to make sure that you execute that code after the initialization on the React Native side.
```

Comment:
```suggestion
Make sure your native code is not executed before the React Native initialization is called. When you initialize the SDK on the React Native side, it is also initialized on the native side. If you write any native code that relies on our SDK, make sure that you execute that code after you initialize the SDK on the React Native side.
```

Reply:
Thanks for the call out. After spending more time with it, I realized how repetitive the original was!